src/locales/en.json

```
@@ -19,6 +19,7 @@
     "opacity": "Opacity",
     "fontSize": "Font Size",
     "fontFamily": "Font Family",
+    "textAlign": "Text Align",
```

```suggestion
    "textAlign": "Text align",
```

We switched to Sentence case